src/index.tsx

```diff
@@ -8,7 +8,7 @@ export interface UseSuperclusterArgument<P, C> {
   points: Array<Supercluster.PointFeature<P>>;
   bounds?: BBox;
   zoom: number;
-  options?: Supercluster.Options<P, C>;
+  options?: Supercluster.Options<P, C> & { disableRefresh?: boolean };
```

Comment:
```suggestion
  options?: Supercluster.Options<P, C>;
  disableRefresh?: boolean;
```

Sorry @wherehows I don't believe I explained it well... I was hoping to see if we could have this param as separate from the supercluster options so we don't have to modify the type of `options` which is a direct passthrough to the supercluster library, rather than something specific for this hook.

Everything about your PR is great, I was just hoping this could be passed in separately from `options`.